java/src/main/native/src/RmmJni.cpp

```diff
@@ -154,13 +154,6 @@ class tracking_resource_adaptor final : public base_tracking_resource_adaptor {
   }
 };
 
-template <typename Upstream>
-tracking_resource_adaptor<Upstream>* make_tracking_adaptor(Upstream* upstream,
```

Comment:
I saw no uses of this function in the `rapidsai` or `NVIDIA` orgs (including `spark-rapids-jni`), so it should be safe to remove.